### ISSUE TITLE
AcquisitionSource and AcquisitionTerms - discrepancy in example usage

### ISSUE THREAD

**Issue by frosterus** [lcnetdev/bibframe-ontology#118]
In the AcquisitionSource example here
https://id.loc.gov/ontologies/bibframe.html#c_AcquisitionSource
there seems to be some strangeness.

```
<bf:Instance rdf:about="http://id.loc.gov/resources/instances/10002730" >
  <bf:acquisitionSource >
    <bf:AcquisitionSource >
      <bf:acquisitionTerms >Romanized</bf:acquisitionTerms>
    </bf:AcquisitionSource>
  </bf:acquisitionSource>
</bf:Instance>
``` 

Original data:
https://id.loc.gov/tools/bibframe/compare-id/full-ttl?find=10002730

First of all, the example seems to contain a mistake in that the 350 subfield a should contain price information but here it reads 'Romanized'. But aside from that, the converter places bf:acquisitionTerms inside bf:AcquisitionSource. In the bf:acquisitionTerms the suggested usage is with Work or Instance and the example there seems to conform to this:
https://id.loc.gov/ontologies/bibframe.html#p_acquisitionTerms

Here is an example where the price information is in 020 field and seems correct:

```
<bf:Instance rdf:about="http://id.loc.gov/resources/instances/1000000" >
<xml:comment >properties truncated from original</xml:comment>
<bf:acquisitionTerms >0.35rub</bf:acquisitionTerms>
</bf:Instance>
``` 

Original data:
https://id.loc.gov/tools/bibframe/compare-id/full-ttl?find=1000000

In summary: the first example seems to be wrong and there seems to be a discrepancy in the Suggested Use values.

**Comment by kirkhess** [comment]
And 037 - Source of Acquisition.  

https://github.com/lcnetdev/marc2bibframe2/blob/cdff84cf049834bb6f9035a773b1b8a9d4935e41/xsl/ConvSpec-010-048.xsl#L1426-L1519

**Comment by kirkhess** [comment]
It seems like the whole concept is obsolete? In BFAS it says "Generally do not enter terms of availability." 
https://www.oclc.org/bibformats/en/0xx/020.html#subfieldc

**Comment by jodiw01** [comment]
Thanks for the feedback. The question about the examples led to a review of the marc2bibframe conversion, and a future update will include these changes:
1 - field 350 will not be converted
2 - acquisitions information in 020 $c and 024 $c will be converted as part of the bf:identifiedBy block. Sample:
```
<bf:identifiedBy>
<bf:Isbn>
<rdf:value>0939416492</rdf:value>
<bf:qualifier>pbk</bf:qualifier>
<bf:acquisitionTerms>$10.95</bf:acquisitionTerms>
</bf:Isbn>
</bf:identifiedBy>

```

**Comment by jodiw01** [comment]
resolved in marc2bibframe v2.6
